text/083-boost-stdlib-dev/text.md

```diff
@@ -0,0 +1,786 @@
+    FSets/FSetToFiniteSet.v, FSets/FMapAVL.v, FSets/FMapPositive.v,
+    FSets/FSets.v, FSets/FMaps.v, FSets/FSetFacts.v,
+    FSets/FSetEqProperties.v, FSets/FSetBridge.v,
+    FSets/FMapWeakList.v, MSets/MSetList.v, MSets/MSetDecide.v,
+    MSets/MSets.v, MSets/MSetWeakList.v, MSets/MSetGenTree.v,
+    MSets/MSetAVL.v, MSets/MSetInterface.v, MSets/MSetFacts.v,
+    MSets/MSetProperties.v, MSets/MSetRBT.v, MSets/MSetPositive.v,
+    MSets/MSetToFiniteSet.v, MSets/MSetEqProperties.v</details>
+* coq-roots-compat
+  * The Coq standard library, compatibility files
+  * Some compatibility files with older versions. Namespace `Coq.Compat`.
+  * 5 files, 0.1 kloc
+  * <details><summary>contains:</summary> Compat/AdmitAxiom.v,
+    Compat/Coq817.v, Compat/Coq819.v, Compat/Coq818.v,
+    Compat/Coq820.v</details>
+* coq-roots-derive
+  * The Coq standard library, derive
+  * Namespace `Coq.derive`
+  * 1 file, 0.0 kloc
+  * <details><summary>contains:</summary>
+    theories/derive/Derive.v</details>
+* coq-roots-rtauto
+  * The Coq standard library, rtauto
+  * Namespace `Coq.rtauto`
+  * 2 files, 0.8 kloc
+  * <details><summary>contains:</summary> rtauto/Bintree.v,
+    rtauto/Rtauto.v</details>
+* coq-roots-wellfounded
+  * The Coq standard library, wellfounded
+  * Namespace `Coq.wellfounded`
+  * 9 files, 0.9 kloc
+  * <details><summary>contains:</summary> Wellfounded/Inclusion.v,
+    Wellfounded/Wellfounded.v, Wellfounded/Union.v,
+    Wellfounded/Transitive_Closure.v, Wellfounded/Well_Ordering.v,
+    Wellfounded/Inverse_Image.v, Wellfounded/Disjoint_Union.v,
+    Wellfounded/Lexicographic_Product.v,
+    Wellfounded/Lexicographic_Exponentiation.v</details>
+* coq-roots
+  * The Coq standard library
+  * A metapackage for all coq-roots-* packages constituting
+    the historical Coq standard library, that used to be shipped with Coq.
+  * 508 files, 183.4 kloc
+
+theories/ssr and theories/ssrmatching should go back into mathcomp
+(either in coq-mathcomp-ssreflect or in a new coq-mathcomp-ssr
+containing only that).
+
+### Documentation
+
+Need to document the above split in packages (could help structure the doc).
+
+## Maintainers
+
+We should do an open call for maintainers. Current maintainers (stdlib
+team from Coq on github) will become roots maintainer, they can of
+course opt out at any time.
+
+# Drawbacks
+
+The stdlib will no longer come bundled with Coq. It will however be
+just as easy to install via the usual package managers. More notably
+it will remain an integral part of the Coq Platform.
+
+# Alternatives
+
+The technical changes proposed here can stand on their own merits, and can be adopted without ratifying the perspective presented for motivating them. Further, each of the three splits discussed under [Packages](https://github.com/proux01/ceps/blob/boost-stdlib-dev/text/083-boost-stdlib-dev/text.md#packages) could be performed without the other ones, with or without the broader vision that led to them.
```

Comment:
The sentence there reads "each requiring the previous one", which seems contradictory?

Reply:
Yeah, it does. Could you explain are the requirements because of which 2 cannot happen without 1 and 3 cannot happen without 2?

Reply:
Well, 2. in https://github.com/proux01/ceps/blob/boost-stdlib-dev/text/083-boost-stdlib-dev/text.md#packages is exposing the split so it need to exist in the first place (that is 1.) and 3. is repo split so hard to do without exposing that split (2.)